### ISSUE TITLE
Implement new method of visualizing PFAM/ELM Data

### ISSUE THREAD

**Issue by elijahxtai** [wonjunetai/pulse-data-viz#15]
Instead of just displaying elm_1, elm_2, elm_3, elm_4 probabilities, use @ccorbi's script for extracting eukaryotic linear motifs in order to visualize them in a horizontal bar format (to visualize the transcript itself), and create a mapping of the locations of where the ELMs are in the transcript. 

For PFAM - since it gives you the alignment start and end locations, place the PFAM domains on the same transcript visualization as ELM in order to see both as linear transcripts.

**Comment by elijahxtai** [comment]
Hey @ccorbi, I have a new format of storing ELM and PFAM coords. Question - is it possible for a transcript to have multiple of the same ELMs in a protein sequence?

```
{
  "I.\"ENST00000466268\";10_51550041_51550174-35=134=106":{  
      "PFAM":{  
         "PF05825.6": {
            "coordinates:[
               1,
               79
            ]
         }
      },
      "ELM":{  
         "DOC_CKS1_1":{  
            "coordinates":[  
               34,
               40
            ],
            "probability":0.00199146382298
         },
         "TRG_ENDOCYTIC_2":{  
            "coordinates":[  
               56,
               60
            ],
            "probability":0.002587491898
         },
         "LIG_14-3-3_3":{  
            "coordinates":[  
               66,
               72
            ],
            "probability":0.00494796200508
         },
         "LIG_SH2_STAT5":{  
            "coordinates":[  
               56,
               60
            ],
            "probability":0.003295871152
         },
         "MOD_SUMO_for_1":{  
            "coordinates":[  
               49,
               53
            ],
            "probability":0.00191400066329
         },
         "MOD_OFUCOSY":{  
            "coordinates":[  
               18,
               24
            ],
            "probability":4.61694301554e-05
         },
         "LIG_SH2_PTP2":{  
            "coordinates":[  
               56,
               60
            ],
            "probability":0.00024539307209
         },
         "DEG_Nend_UBRbox_2":{  
            "coordinates":[  
               0,
               3
            ],
            "probability":0.00025370818551
         },
         "MOD_PK_1":{  
            "coordinates":[  
               66,
               73
            ],
            "probability":0.000941753592931
         },
         "DOC_USP7_UBL2_3":{  
            "coordinates":[  
               51,
               56
            ],
            "probability":0.0037417689
         }
      }
   }
}
```

**Comment by elijahxtai** [comment]
Commit 9d30eaa472d4813d538b081b05cae668469fd17e adds PFAM into JSON.

**Comment by elijahxtai** [comment]
For reference: ELM data from here: http://elm.eu.org/downloads.html

**Comment by elijahxtai** [comment]
Edited JSON formatting:

```
[  
  {  
      "pfam":[  
         {  
            "pfam_id":"PF00664.18",
            "coordinates":[  
               326,
               596
            ]
         },
         {  
            "pfam_id":"PF00005.22",
            "coordinates":[  
               661,
               795
            ]
         },
         {  
            "pfam_id":"PF00664.18",
            "coordinates":[  
               972,
               1246
            ]
         },
         {  
            "pfam_id":"PF00005.22",
            "coordinates":[  
               1310,
               1423
            ]
         }
      ],
      "entry":"I.\"ENST00000345148\";16_16177223_16177399-127=177=168",
      "elms":[  
         {  
            "elm_id":"LIG_OCRL_FandH_1",
            "coordinates":[  
               329,
               342
            ],
            "probability":9.77240676559e-06
         },
         {  
            "elm_id":"LIG_PAM2_1",
            "coordinates":[  
               1188,
               1201
            ],
            "probability":9.95389535837e-06
         },
         {  
            "elm_id":"DOC_PP1_SILK_1",
            "coordinates":[  
               1053,
               1059
            ],
            "probability":3.77750556097e-05
         },
         {  
            "elm_id":"DEG_ODPH_VHL_1",
            "coordinates":[  
               101,
               115
            ],
            "probability":7.03776543919e-05
         },
         {  
            "elm_id":"DOC_SPAK_OSR1_1",
            "coordinates":[  
               1130,
               1135
            ],
            "probability":7.67825867606e-05
         },
         {  
            "elm_id":"LIG_MYND_3",
            "coordinates":[  
               746,
               751
            ],
            "probability":8.4443906863e-05
         },
         {  
            "elm_id":"LIG_MAD2",
            "coordinates":[  
               273,
               282
            ],
            "probability":0.000101063469513
         },
         {  
            "elm_id":"CLV_MEL_PAP_1",
            "coordinates":[  
               1071,
               1078
            ],
            "probability":0.000105375572148
         },
         {  
            "elm_id":"LIG_AP2alpha_2",
            "coordinates":[  
               1388,
               1391
            ],
            "probability":0.000119206092212
         },
         {  
            "elm_id":"LIG_eIF4E_1",
            "coordinates":[  
               175,
               182
            ],
            "probability":0.000189127545301
         },
         {  
            "elm_id":"LIG_Pex14_1",
            "coordinates":[  
               39,
               44
            ],
            "probability":0.000222625
         },
         {  
            "elm_id":"LIG_SH2_PTP2",
            "coordinates":[  
               567,
               571
            ],
            "probability":0.00024539307209
         },
         {  
            "elm_id":"LIG_NRBOX",
            "coordinates":[  
               75,
               82
            ],
            "probability":0.000262764536293
         },
         {  
            "elm_id":"MOD_N-GLC_2",
            "coordinates":[  
               1296,
               1299
            ],
            "probability":0.000297299203491
         },
         {  
            "elm_id":"LIG_TYR_ITIM",
            "coordinates":[  
               1123,
               1129
            ],
            "probability":0.000299240952879
         },
         {  
            "elm_id":"LIG_PTB_Apo_2",
            "coordinates":[  
               22,
               30
            ],
            "probability":0.000310820773429
         },
         {  
            "elm_id":"LIG_BIR_II_1",
            "coordinates":[  
               0,
               5
            ],
            "probability":0.000325201831485
         },
         {  
            "elm_id":"LIG_Clathr_ClatBox_1",
            "coordinates":[  
               122,
               127
            ],
            "probability":0.000340550653717
         },
         {  
            "elm_id":"LIG_Pex14_2",
            "coordinates":[  
               28,
               33
            ],
            "probability":0.000462812031
         },
         {  
            "elm_id":"LIG_SH2_GRB2",
            "coordinates":[  
               439,
               443
            ],
            "probability":0.000478686991
         },
         {  
            "elm_id":"LIG_Actin_WH2_2",
            "coordinates":[  
               523,
               542
            ],
            "probability":0.000660304959775
         },
         {  
            "elm_id":"TRG_NES_CRM1_1",
            "coordinates":[  
               171,
               185
            ],
            "probability":0.000762613786385
         },
         {  
            "elm_id":"DEG_APCC_DBOX_1",
            "coordinates":[  
               591,
               600
            ],
            "probability":0.000767697345968
         },
         {  
            "elm_id":"LIG_SH2_STAT3",
            "coordinates":[  
               709,
               713
            ],
            "probability":0.000797495836
         },
         {  
            "elm_id":"DOC_PP1_RVXF_1",
            "coordinates":[  
               168,
               176
            ],
            "probability":0.000830144384932
         },
         {  
            "elm_id":"MOD_PK_1",
            "coordinates":[  
               1141,
               1148
            ],
            "probability":0.000941753592931
         },
         {  
            "elm_id":"LIG_UBA3_1",
            "coordinates":[  
               122,
               130
            ],
            "probability":0.00119615886678
         },
         {  
            "elm_id":"MOD_NEK2_2",
            "coordinates":[  
               608,
               614
            ],
            "probability":0.00129503438775
         },
         {  
            "elm_id":"LIG_14-3-3_2",
            "coordinates":[  
               414,
               421
            ],
            "probability":0.00155688870378
         },
         {  
            "elm_id":"LIG_TRAF6",
            "coordinates":[  
               232,
               241
            ],
            "probability":0.00171471882779
         },
         {  
            "elm_id":"LIG_BRCT_BRCA1_1",
            "coordinates":[  
               193,
               198
            ],
            "probability":0.001911520721
         },
         {  
            "elm_id":"MOD_SUMO_for_1",
            "coordinates":[  
               938,
               942
            ],
            "probability":0.00191400066329
         },
         {  
            "elm_id":"MOD_CDK_1",
            "coordinates":[  
               300,
               307
            ],
            "probability":0.00192871947029
         },
         {  
            "elm_id":"DOC_CKS1_1",
            "coordinates":[  
               65,
               71
            ],
            "probability":0.00199146382298
         },
         {  
            "elm_id":"DOC_PP2B_LxvP_1",
            "coordinates":[  
               474,
               478
            ],
            "probability":0.00229635088095
         },
         {  
            "elm_id":"MOD_PKA_1",
            "coordinates":[  
               414,
               421
            ],
            "probability":0.0023150946744
         },
         {  
            "elm_id":"LIG_SUMO_SIM_anti_2",
            "coordinates":[  
               179,
               185
            ],
            "probability":0.00234949254565
         },
         {  
            "elm_id":"TRG_ENDOCYTIC_2",
            "coordinates":[  
               63,
               67
            ],
            "probability":0.002587491898
         },
         {  
            "elm_id":"TRG_LysEnd_APsAcLL_1",
            "coordinates":[  
               33,
               39
            ],
            "probability":0.00275767155936
         },
         {  
            "elm_id":"CLV_C14_Caspase3-7",
            "coordinates":[  
               288,
               293
            ],
            "probability":0.00309374033071
         },
         {  
            "elm_id":"LIG_SH2_STAT5",
            "coordinates":[  
               44,
               48
            ],
            "probability":0.003295871152
         },
         {  
            "elm_id":"LIG_LIR_Apic_2",
            "coordinates":[  
               664,
               669
            ],
            "probability":0.00337096434764
         },
         {  
            "elm_id":"DOC_USP7_UBL2_3",
            "coordinates":[  
               258,
               263
            ],
            "probability":0.0037417689
         },
         {  
            "elm_id":"CLV_PCSK_PC1ET2_1",
            "coordinates":[  
               613,
               616
            ],
            "probability":0.00390276834
         },
         {  
            "elm_id":"LIG_TRAF2_1",
            "coordinates":[  
               292,
               296
            ],
            "probability":0.00429981271202
         },
         {  
            "elm_id":"DOC_MAPK_1",
            "coordinates":[  
               71,
               79
            ],
            "probability":0.00432380001106
         },
         {  
            "elm_id":"LIG_SUMO_SIM_par_1",
            "coordinates":[  
               557,
               564
            ],
            "probability":0.00454520100697
         },
         {  
            "elm_id":"LIG_14-3-3_3",
            "coordinates":[  
               485,
               491
            ],
            "probability":0.00494796200508
         }
      ]
   }
]
```

**Comment by wonjunetai** [comment]
Closed with #17

**Comment by elijahxtai** [comment]
Addressed by #17
